lib/common.sh

```diff
@@ -308,7 +308,13 @@ fi
 export KIND_NODE_IMAGE=${KIND_NODE_IMAGE:-"${DOCKER_HUB_PROXY}/kindest/node:${KIND_NODE_IMAGE_VERSION}"}
 
 # Ansible version
-export ANSIBLE_VERSION=${ANSIBLE_VERSION:-"7.0.0"}
+# Older ubuntu version do no support 7.0.0 because of older python versions
+# Ansible 7.0.0 requires python 3.10+
+if [[ ${DISTRO} == "ubuntu22" ]]; then
```

Comment:
[[ is unnecessary here as string comparison is basic sh functionality.

[ "${DISTRO}" = "ubuntu22" ] is more portable.

Reply:
There are many instances of double [[ and single [ in this file for string comparison. does it make any difference? or we can keep them as it is.

Reply:
Shellcheck fails if  I do that @tuminoid , keeping it as it is.

Reply:
AH no sorry, shellcheck was saying to double quote

Reply:
Changed.

Reply:
As @smoshiur1237 mentioned, the convention was so far to use double brackets most of the places in this file and thoughout the repo, we need to start changing it as some point otherwise mixed conventions would create confusion.